### PR TITLE
Update copy-into-transact-sql.md

### PR DESCRIPTION
**Pull request by jasonhorner**
added a section to help point people to some resources for resolving: HdfsBridge::recordReaderFillBuffer Error it would be really good to add some additional information from the product team on how to troubleshoot these there are a few variants and I only address one here.

### CODE REVIEW

File: docs/t-sql/statements/copy-into-transact-sql.md [hunk 1]
```diff
@@ -526,7 +526,11 @@ Follow these steps to work around this issue by re-registering the workspace's m
    Select-AzSubscription -SubscriptionId <subscriptionId>
    Set-AzSqlServer -ResourceGroupName your-database-server-resourceGroup -ServerName your-SQL-servername -AssignIdentity
    ```
-   
+## Resolving HdfsBridge::recordReaderFillBuffer Errors
+You may experience the error HdfsBridge::recordReaderFillBuffer if types are mismatched between Parquet and SQL or if you have unsupported Parquet data types.
+Check this link to verify the that column names, order, and datatypes are correctly mapped: [Type Mapping for Parquet](https://learn.microsoft.com/en-us/azure/synapse-analytics/sql/develop-openrowset#type-mapping-for-parquet) 
+a quick way to resolve these or further troubleshoot is to load with AUTO_CREATE_TABLE = ON or create a table with all data types set to NVARCHAR.
```
Comment (Court72) [inline docs/t-sql/statements/copy-into-transact-sql.md]:
```suggestion
You can experience the error `HdfsBridge::recordReaderFillBuffer` if types are mismatched between Parquet and [!INCLUDE[ssazuresynapse-md](../../includes/ssazuresynapse-md.md)], or if you have unsupported Parquet data types.

Verify that column names, order, and data types are correctly mapped, using the information in [Type mapping for Parquet](/azure/synapse-analytics/sql/develop-openrowset#type-mapping-for-parquet).

A quick way to resolve or to further troubleshoot this issue is to load with `AUTO_CREATE_TABLE = ON`, or create a table with all data types set to `NVARCHAR`. 
```

**Comment by Court72** [comment]
@periclesrocha

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/t-sql/statements/copy-into-transact-sql.md [hunk 2]
```diff
@@ -526,7 +526,11 @@ Follow these steps to work around this issue by re-registering the workspace's m
    Select-AzSubscription -SubscriptionId <subscriptionId>
    Set-AzSqlServer -ResourceGroupName your-database-server-resourceGroup -ServerName your-SQL-servername -AssignIdentity
    ```
-   
+## Resolving HdfsBridge::recordReaderFillBuffer Errors
```
Comment (rwestMSFT) [inline docs/t-sql/statements/copy-into-transact-sql.md]:
```suggestion
## Resolve HdfsBridge::recordReaderFillBuffer errors

```

File: docs/t-sql/statements/copy-into-transact-sql.md [hunk 3]
```diff
@@ -526,7 +526,14 @@ Follow these steps to work around this issue by re-registering the workspace's m
    Select-AzSubscription -SubscriptionId <subscriptionId>
    Set-AzSqlServer -ResourceGroupName your-database-server-resourceGroup -ServerName your-SQL-servername -AssignIdentity
    ```
-   
+## Resolve HdfsBridge::recordReaderFillBuffer errors
```
Comment (rwestMSFT) [inline docs/t-sql/statements/copy-into-transact-sql.md:530]:
```suggestion

## Resolve HdfsBridge::recordReaderFillBuffer errors
```

**Comment by periclesrocha** [comment]
Hello @jasonhorner, thanks for your contribution. This problem, however, is the product of a user error (data type mapping), whereas here we outline known issues in the platform. I see this error is well covered in articles on the internet. I recommend not approving this PR for these reasons. If you feel strongly otherwise, please let me know.

**Comment by rwestMSFT** [comment]
@jasonhorner We haven't heard from you yet, but I'm closing this PR in the meantime. If you have any comments, feel free to comment below.

**Comment by jasonhorner** [comment]
@periclesrocha  cc @rwestMSFT Thank you for your feedback. My intent with this PR is to address the challenge users face in real-world scenarios, where the current error message may not be sufficient. I'm open to specific suggestions for refining or improving the PR and happy to devote the time to fix it.     

In cases like a table with 30 columns, the existing message lacks clarity and can be extremely frustrating to troubleshoot. The abundance of internet discussions on this issue may indicate a gap in official documentation, prompting my PR to enhance user guidance. I plan on opening a support ticket to solve a specific issue we are seeing and hopefully improve the error messaging (I feel like this is a very similar scenario to the reasons VERBOSE_TRUNCATION_WARNINGS was introduced in 2019)

I understand the error isn't directly from the "copy into" command, but I do feel strongly that the existing documentation should be improved to provide better guidance to resolve this.

**Comment by rwestMSFT** [comment]
@periclesrocha This type of thing might be better suited to the troubleshooting repo, not sql-docs.

**Pull request closed without merging**